source/MRMesh/MRVolumeSegment.cpp

```diff
@@ -211,7 +211,11 @@ tl::expected<VoxelBitSet, std::string> VolumeSegmenter::segmentVolume( float seg
     }
 
     // Segment volume
-    return segmentVolumeByGraphCut( volumePart_, segmentationExponentModifier, seedsInVolumePartSpace_[Inside], seedsInVolumePartSpace_[Outside] );
+    auto res = segmentVolumeByGraphCut( volumePart_, segmentationExponentModifier, seedsInVolumePartSpace_[Inside], seedsInVolumePartSpace_[Outside], cb );
```

Comment:
Can it be empty if segmentation failed because of some other reason (not user cancelation)?